app/src/main/res/values/strings.xml

```diff
@@ -97,4 +97,5 @@
     <string name="no_current_mentorship_relation">You are not in a current mentorship relation.</string>
     <string name="notes_empty_error">Notes cannot be empty</string>
     <string name="available_to_be_mentor_mentee">Available to be a:</string>
+    <string name="cannot_send_request">Sorry, the user is neither available to be a mentor nor he/she needs mentoring!!</string>
```

Comment:
"This user is not available to be a mentor nor a mentee."

Reply:
"This user is not available to be a mentor or a mentee"

Reply:
Follow @m-murad on this 😝